contracts/gamecontract/Gamecontract.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+
+contract GameContract is Ownable {
+
+    error GameContract__NotAuthorized();
+
+    mapping(address => bool) public signers;
+
+    event DailyQuestSubmit(address indexed user, string questName);
+    event SocialQuestSubmit(address indexed user, string questName);
+    event GameQuestSubmit(address indexed user, string questName);
+
+    modifier User() {
+        if (!signers[msg.sender]) {
+            revert GameContract__NotAuthorized();
+        }
+        _;
+    }
+
+    constructor() {}
```

Comment:
Empty construction?

Reply:
yes i will remove it because i use library ownable so the deployer contract will always be the owner contract